src/pages/groupLifeCycleLog.vue

```diff
@@ -0,0 +1,56 @@
+<script setup>
+import { onMounted, ref } from 'vue';
+import { fetchGroupLifeCycleLogs } from "../api/groupLifeCycleLog";
+import { FilterMatchMode } from "primevue/api";
+
+
+
+   const logs = ref([]);
+   const filters = ref({
+      global: { value: null, matchMode: FilterMatchMode.CONTAINS },
+   });
+
+   const fetchLogs = async () => {
```

Comment:
Make use of try catches when making API calls. And use await instead of .then
Make the calls easier to handle and more readable

const fetchLogs = async () => {
  try {
    const data = await fetchGroupLifeCycleLogs();
    logs.value = data;
  } catch (error) {
    console.error("Error fetching logs:", error);
  }
};

Reply:
> Make use of try catches when making API calls. And use await instead of .then Make the calls easier to handle and more readable
> 
> const fetchLogs = async () => { try { const data = await fetchGroupLifeCycleLogs(); logs.value = data; } catch (error) { console.error("Error fetching logs:", error); } };

Should the try catch in `fetchGroupLifeCycleLogs()` function in api/groupLifeCycleLog.js then be removed in replacement of this one then or should they both be there?

Reply:
Ah right, I missed it because it was commented out. No you are correct, the try catch should be in the API call